### PR TITLE
Add section meetings

### PR DESCRIPTION
**Pull request by hamiltomamede**
Add section meeting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,3 +35,10 @@ This is best team in the world you could ever find
 - Ruby on Rails
 - Kendo
 - JS 
+
+# Meetings
+
+### Scrum meetings
+
+- [refinement meeting](refinement.md)
```
Comment (ianpierreg) [inline README.md]: Lower case

File: refinement.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Refinement Meeting
```
Comment (Marinofull) [inline refinement.md]:
```suggestion
## Refinement Meeting
```

File: README_BACKUP_19314.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+## DreamTeam
```
Comment (Marinofull) [inline README_BACKUP_19314.md]: there are more three files created in the conflicts to be removes

File: refinement.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Refinement Meeting
+Scrum meeting where the team refine the histories.
+
+### What time?
+01:00PM
+
+### How long?
+Depends of the histories
+
```
Comment (Marinofull) [inline refinement.md]: extra blank lines to be removed

**Pull request merged**